devdox/tests/unit_test/app/services/test_git_tokens_service.py

```
@@ -61,3 +61,75 @@ async def test_bubbles_up_store_exception(self):
             )
 
         assert str(exc.value) == "Boom"
+
+@pytest.mark.asyncio
+class TestGetGitLabelService:
+    
```

Flake8 (7.2.0)

[error] indentation contains mixed spaces and tabs

(E101)